tests/Gedmo/Mapping/Annotation/TranslatablePropertyTestCase.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gedmo\Tests\Mapping\Annotation;
+
+use Gedmo\Mapping\Annotation\Translatable;
+use Gedmo\Tests\Mapping\Fixture\Annotation\TranslatableModel as AnnotationTranslatableModel;
+use Gedmo\Tests\Mapping\Fixture\Attribute\TranslatableModel as AttributeTranslatableModel;
+
+class TranslatablePropertyTestCase extends BasePropertyAnnotationTestCase
```

Comment:
Is this class intended to be extended? Otherwise, I think we can mark it as final.